.github/workflows/tenzir.yaml

```diff
@@ -609,6 +609,9 @@ jobs:
             cxx: g++-12
             dependencies-script-path: scripts/debian/install-dev-dependencies.sh
             cmake-extra-flags: -DTENZIR_ENABLE_BUNDLED_CAF:BOOL=ON
+            bundled-plugins:
+              - plugins/web
+              - plugins/fluent-bit
```

Comment:
I know why for the web plugin, but why is the fluent-bit plugin bundled?

Reply:
I forgot to move the comment around.